### PR TITLE
Commenting on folders

### LINKED ISSUE #2905: Update commenting section
https://learning.postman.com/docs/collaborating-in-postman/commenting-on-collections/

The UX for adding a comment to a collection has changed a bit with V8 / context bar.
You can add comments to folders now too.
Might be helpful to clarify when comments are associated with a user (vs private user)

### PR DESCRIPTION
**Pull request by sowmyaraj92**
To resolve #2905

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/commenting-on-collections.md [hunk 1]
```diff
@@ -50,32 +53,42 @@ You can add comments at the [request level](#adding-a-comment-on-a-request), on
...
+5. Enter your comment, then click **Add Comment**.
+
+![Add inline comments for tests](https://assets.postman.com/postman-docs/adding-an-inline-comment-tests-v8.gif)
+
+## Commenting on a folder
+
+You can leave comments on folders in Postman.
+
+1. Open the folder you would like to comment on.
+2. Open the **Comments** icon <img alt="Mini comments icon" src="https://assets.postman.com/postman-docs/mini-comments-icon-v8.jpg" width="20px"/> at the top-right of the context bar.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/commenting-on-collections.md:88]: I'd intro this by combining it with the similar command "2. Switch to **Comment** mode at the top-right of the window." in the other lists like:

"Switch to **Comment** mode by selecting the **Comments** icon Mini comments icon at the top-right of the context bar."

And update that step in other sections to be the same if it makes sense

File: src/pages/docs/collaborating-in-postman/commenting-on-collections.md [hunk 2]
```diff
@@ -94,24 +107,24 @@ You can make changes to comments you've already posted. Other team members canno
...
-3. Click **Resolve** next to the comment(s) you'd like to resolve.
+3. Click **Resolve** next to the comment(s) you would like to resolve.
 
-![Resolve comments](https://assets.postman.com/postman-docs/resolve-comments-v8.gif)
+![Resolve comments](https://assets.postman.com/postman-docs/resolving-a-comment-v8.gif)
 
 ## Deleting comments
 
 1. In Postman, open the collection with the comment you want to delete.
 2. Find the comment and click **Comments** > **Delete**.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/commenting-on-collections.md:125]: I'd switch this instruction to clicking the trash can icon

**Review by claireriley — APPROVED** [review]
LGTM! Added a couple suggestions, but approving as it's ready to go

File: src/pages/docs/collaborating-in-postman/commenting-on-collections.md (hunk 1 above)
Comment (sowmyaraj92) [reply src/pages/docs/collaborating-in-postman/commenting-on-collections.md:88]: We have **Comment mode** only for requests. For folders, we have to open the comments icon from the context bar.

**Pull request merged**